build.js

```diff
@@ -35,18 +36,32 @@ var isOptional = function isOptional(category) {
   return (category || '').indexOf('annex b')>-1 || category === 'pre-strawman' || category === 'strawman (stage 0)';
 };
 
+var byTestSuite = function(suite) {
+  return function(browser) {
+    return Array.isArray(browser.test_suites) ? browser.test_suites.indexOf(suite)>-1 : true;
```

Comment:
No, I mean exactly `>-1` as pre-ES2017 alternative to `Array.prototype.includes`, using same codestyle as anywhere else in this file